Dockerfile

```
@@ -0,0 +1,47 @@
+# This builds Boulder in a Docker container, then creates an image
+# containing just the built Boulder binaries plus some ancillary
+# files that are useful for predeployment testing.
+FROM docker.io/ubuntu:24.04 AS builder
+
+ARG COMMIT_ID
+ARG GO_VERSION
+ARG VERSION
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get --assume-yes --no-install-recommends --update install \
+    ca-certificates curl gcc git gnupg2 libc6-dev
```

Purging the apt cache here can save us a little on the size of this layer.

```suggestion
RUN apt-get --assume-yes --no-install-recommends --update install \
        ca-certificates \
        curl \
        gcc \
        git \
        gnupg2 \
        libc6-dev && \
    rm -rf /var/lib/apt/lists/*

```